djstripe/management/commands/_djstripe_command_utils.py

```diff
@@ -0,0 +1,32 @@
+import sys
```

Comment:
Just to confirm, this mixin is specific for management commands. The file pointed to here looks like it's for more Stripe model stuff in general. Are you sure you still want it moved there @kavdev? Also, opinions on removing it altogether since the commands are being squashed into one?

Reply:
I think it's ok in mixins.py - it would be useful for other commands.